lib/ansible/modules/packaging/os/apt.py

```diff
@@ -133,6 +134,7 @@
      (If you typo C(foo) as C(fo) apt-get would install packages that have "fo" in their name with a warning and a prompt for the user.
      Since we don't have warnings and prompts before installing we disallow this.Use an explicit fnmatch pattern if you want wildcarding)
    - When used with a `loop:` each package will be processed individually, it is much more efficient to pass the list directly to the `name` option.
+   - The deb parameter require xz-utils package in order to extract the control file of the deb package to install.
```

Comment:
I'd put this line under the current line 95, in the description of the `deb` parameter.

Reply:
Yeah i wasn't sure for both point. Thanks.